Src/ILGPU/Context.cs

```diff
@@ -170,9 +171,10 @@ public Context(ContextFlags flags, OptimizationLevel optimizationLevel)
                 : new ILFrontend(frontendDebugInformationManager, 1);
 
             // Create default IL backend
+            var cpuCapabilities = new CPUCapabilityContext();
```

Comment:
Question: Do we really need to pass a `CPUCapabilityContext` instance to each `ILBackend` instance?

Reply:
Changed to have the `CPUCapabilityContext` constructed internally in the `ILBackend`.